client/src/actions/auth.js

```diff
@@ -28,3 +37,54 @@ export const signUp =
 			dispatch({ type: REGISTER_FAIL });
 		}
 	};
+
+// // login user
+// export const login = (email, password) => async dispatch => {
+// 	const config = {
+// 		headers: {
+// 			'Content-Type': 'application/json',
+// 		},
+// 	};
+// 	const body = JSON.stringify({ email, password });
+// 	try {
+// 		const res = await axios.post('/app/auth', body, config);
+
+// 		dispatch({
+// 			type: LOGIN_SUCCESS,
+// 			payload: res.data,
+// 		});
+// 		dispatch(loadUser());
+// 	} catch (err) {
+// 		const errors = err.response.data.errors;
+
+// 		if (errors) {
+// 			errors.forEach(error => dispatch(setAlert(error.msg)));
+// 		}
+// 		dispatch({ type: LOGIN_FAIL });
+// 	}
+// };
+
```

Comment:
I comment this out because, there needs some modification for the login component in order to use this.